longevity_sla_test.py

```diff
@@ -35,72 +29,26 @@ def __init__(self, *args):
         self.roles = []
 
     def test_custom_time(self):
-        with self.db_cluster.cql_connection_patient(node=self.db_cluster.nodes[0], user=self.DEFAULT_USER,
-                                                    password=self.DEFAULT_USER_PASSWORD) as session:
+        with self.db_cluster.cql_connection_patient(node=self.db_cluster.nodes[0], user=DEFAULT_USER,
+                                                    password=DEFAULT_USER_PASSWORD) as session:
             # Add index (shares position in the self.service_level_shares list) to role and service level names to do
             # it unique and prevent failure when try to create role/SL with same name
             for index, shares in enumerate(self.service_level_shares):
-                self.roles.append(self.create_sla_auth(session=session, shares=shares, index=index))
+                self.roles.append(create_sla_auth(session=session, shares=shares, index=index))
 
             if self.params.get("run_fullscan"):
-                self.fullscan_role = self.create_sla_auth(session=session, shares=self.FULLSCAN_SERVICE_LEVEL_SHARES,
-                                                          index=0)
+                self.fullscan_role = create_sla_auth(session=session, shares=self.FULLSCAN_SERVICE_LEVEL_SHARES,
+                                                     index=0)
+
+        try:
+            LoaderUtilsMixin.add_sla_credentials_to_stress_cmds(
```

Comment:
why not `self.add_sla_credentials_to_stress_cmds` ? i.e. that's the whole point of mixing